common-npm-packages/docker-common-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-docker-common-v2",
-  "version": "1.177.0",
+  "version": "2.0.0-preview.0",
```

Comment:
Let's change it to 3.0.0-preview.0 if the approach of using `azure-pipelines-tasks-docker-common` is confirmed.

cc: @anatolybolshakov